plugin-server/src/worker/vm/upgrades/historical-export/export-historical-events-v2.ts

```diff
@@ -252,6 +253,12 @@ export function addHistoricalEventsExportCapabilityV2(
             return
         }
 
+        if (params.abortMessage) {
+            // For manually triggering the export to abort
+            await stopExport(params, `Export aborted ${params.abortMessage}`, 'fail')
```

Comment:
```suggestion
            await stopExport(params, `Export stopped by instance operator: ${params.abortMessage}`, 'fail')
```

Reply:
we might add a UI button in the future - this should be inside abortMessage